modules/microshift-exposed-audit-ports-hostnetwork.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-exposed-audit-ports-hostnetwork_{context}"]
+
+== hostnetwork
+
+When a pod is configured with the `hostNetwork:true` setting, the pod is running in the host network namespace. This configuration can independently open host ports. {microshift-short} component logs cannot be used to track this case, the ports are subject to firewalld rules. If the port opens in firewalld, you can view the port opening in the firewalld debug log.
+
+.Prerequisites 
+
+* You have root user access to your build host.
+
+.Procedure
+
+. Optional: You can check that the `hostnetwork:true` parameter is set in your ovnkube-node pod by running the following command
++
+[source,terminal]
+----
+$ sudo oc get pod -n openshift-ovn-kubernetes ovnkube-node-7mmlp -o json | jq -r '.spec.hostNetwork' true
```

Comment:
This should be noted as an example only. Not every system will have the exact value `ovnkube-node-7mmlp` since it is just a name. It varies per custer.